script/Arcueid/Day 8/08_00_ARC08_8_2.txt

```diff
@@ -38,12 +38,12 @@ Mmmhmm, that's just how a handsome young man in the humanities should behave!
 [sha:c556a720467b8d0691a188dc04f54c588d9f998a]{
 -- Page 583, Offset 21248.
 -- 吸血鬼と毎夜あんなコトしてるクセに、かーわいー☆」
-Even though you do those %{i}things%{/i} with that vampire every night, how cute ☆"
+Even though you do those %{i}things%{/i} with that vampire every night, how cute ☆."
 }
 [sha:2cceb3713773a5eb558bbca0c71fe9642cd7519e]{
 -- Page 584, Offset 21249. Mods: @n.
 -- 二歩。三歩。四歩。
-Two steps. Three steps. Four steps.
+Two steps. Three steps. Four steps. 
```

Comment:
You sure? it's glued to the line after it (four steps.---)

Reply:
Hm, it rendered on a new line for me in game. I think it might be a scripted break.
Also, if you do need to add spacing for glue, you have to either
- do it on the front of the line that is being glued or
- use a trailing `%{s}` escape code. 
Any right-handed trailing spaces are purged by the parser